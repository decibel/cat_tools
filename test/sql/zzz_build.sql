\set ECHO none

\i test/pgxntool/psql.sql
\t

BEGIN;
CREATE SCHEMA cat_tools;

\i sql/cat_tools.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
