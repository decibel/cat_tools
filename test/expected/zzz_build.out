\set ECHO none
 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

 create_function 
-----------------
 
(1 row)

# TRANSACTION INTENTIONALLY LEFT OPEN!
